### PR TITLE
test(TypeScript): add tests for endpoint class-like wrappers

### PR DESCRIPTION
**Pull request by platosha**
Connected to vaadin/flow#7176
Depends on vaadin/flow#9972

### CODE REVIEW

File: vaadin-spring-tests/test-ts-services/frontend/src/test-component.js [hunk 1]
```diff
@@ -2,13 +2,15 @@ import { PolymerElement } from '@polymer/polymer/polymer-element.js';
...
 import * as packagePrivateEndpoint from '../generated/PackagePrivateEndpoint';
+// import {AppEndpoint} from "../generated/AppEndpoint";
 
 class TestComponent extends PolymerElement {
   static get template() {
     return html`
         <button id="button">vaadin hello</button><br/>
         <button id="hello" on-click="hello">endpoint hello</button><br/>
         <button id="helloAnonymous" on-click="helloAnonymous">endpoint helloAnonymous</button><br/>
+        <button id="helloAnonymousWrapper" on-click="helloAnonymousWrapper">endpoint AppEndpoint.helloAnonymous</button><br/>
```
Comment (haijian-vaadin) [inline vaadin-spring-tests/test-ts-services/frontend/src/test-component.js:13]: Forgot to remove the `endpoint `?

**Review by haijian-vaadin — CHANGES_REQUESTED** [review]

File: vaadin-spring-tests/test-ts-services/frontend/src/test-component.js (hunk 1 above)
Comment (platosha) [reply vaadin-spring-tests/test-ts-services/frontend/src/test-component.js:13]: The button name should not matter, but forgot to remove the import comment. The test was failing to compile.

Fixed.

**Comment by haijian-vaadin** [comment]
Should also bump the Flow version to pass the validation

**Comment by platosha** [comment]
@haijian-vaadin rebased on master, which depends on 7.0-SNAPSHOT Flow now

**Review by haijian-vaadin — APPROVED** [review]

**Pull request merged**